src/PlanktonIndividualExample.jl

```diff
@@ -0,0 +1,1280 @@
+### A Pluto.jl notebook ###
+# v0.15.0
+
+using Markdown
+using InteractiveUtils
+
+# ╔═╡ 153caed2-f735-4508-8e80-f0461a6af88c
+using PlanktonIndividuals, Plots
+
+# ╔═╡ 418c95a7-0125-40b6-acb5-e2011ea60df2
+md"""
+# [PlanktonIndividuals.jl](https://github.com/JuliaOcean/PlanktonIndividuals.jl) tutorial
+"""
+
+
+# ╔═╡ 2b46fec5-59bf-4d5b-8af6-0bd695ac1821
+md"""
+PlanktonIndividuals.jl is a fast individual-based model written in Julia that runs on both CPU and GPU. 
+
+It simulates the life cycle of ocean phytoplankton cells as Lagrangian particles while nutrients are represented as Eulerian tracers and advected over the gridded domain. 
+
+
+![phytoplankton](https://github.com/JuliaOcean/PlanktonIndividuals.jl/raw/master/docs/src/PI_Quota.jpeg)
+"""
+
+# ╔═╡ 6ec492fc-d622-11eb-2bfb-9d24a91685b7
+md"""
+Here we simulate phytoplankton cells as Lagrangian particles in a 2D flow field, with 
+one horizontal direction (x) and one vertical one (z), like in an ocean transect.
+
+The domain is periodic in the x direction while it is bounded in the z direction.
+"""
+
+# ╔═╡ 139108e4-0829-4710-8419-dc05808f173f
+md"""
+## 1. Import packages
+"""
+
+# ╔═╡ 15bc275c-7d15-479c-b644-d588e67cf3f5
+md"""
+Full documentation is available at [https://juliaocean.github.io/PlanktonIndividuals.jl/dev/](https://juliaocean.github.io/PlanktonIndividuals.jl/dev/)
+"""
+
+# ╔═╡ 3c05c373-af76-4d28-979c-e91aaa7321de
+p=dirname(pathof(PlanktonIndividuals));
+
+# ╔═╡ b7f4cd3e-78aa-44cd-bb6c-3f41ed9f8a96
+include(joinpath(p,"../examples/helper_functions.jl"));
+
+# ╔═╡ 37f42666-276f-43bf-b4b4-a245af7edb29
+md"""
+## 2. Architecture, Grid & Flow Fields
+"""
+
+# ╔═╡ 49c16379-d570-4849-bb74-0b7ccedb86a5
+md"""
+First we generate grid information (128 by 128 grid boxes, 1m thick, and 1m wide) and the computational architecture (CPU).
+
+[Speed-ups](https://juliaocean.github.io/PlanktonIndividuals.jl/dev/benchmarks/) on GPU.
+"""
+
+# ╔═╡ 6e45867e-e12e-4e44-9795-0a9ee01ecccb
+arch = CPU()
```

Comment:
This is probably a stupid question, but can I use the GPU on my laptop? I tried and it threw with

```
Cannot create a GPU model. No CUDA-enabled GPU was detected!
```

(Again, I have never used GPUs — intentionally at least — so maybe I'm just out of my depth here!)

Reply:
In order to use GPU mode, you need a CUDA compatible GPU (NVIDIA CUDA GPUs), Mac doesn't have this kind of GPU.